lib/actions/help.coffee

```diff
@@ -78,7 +79,7 @@ command = (params, options, done) ->
 		return done(error) if error?
 
 		if not command? or command.isWildcard()
-			return done(new Error("Command not found: #{params.command}"))
+			expectedError("Command not found: #{params.command}")
```

Comment:
I was tricked by this for a bit, not knowing that `expectedError` calls `process.exit` :)

Reply:
@thgreasi Hmm, interesting, and a good point. We should definitely have a verb in here. How do you feel about `failWithExpectedError`?

Reply:
I've gone for `exitWithExpectedError` - I'd rather this be a little verbose but clearer I think.

Thanks for bringing this up! Good improvement :+1: